src/main/java/rs/hybridit/dto/BookCopyDto.java

```diff
@@ -0,0 +1,56 @@
+package rs.hybridit.dto;
+
+import java.util.Date;
+import rs.hybridit.model.Book;
+
+public class BookCopyDto {
+
+	private Long id;
+	private Date rentStart;
+	private Date rentEnd;
+	private Book book;
+
+	public BookCopyDto(){
```

Comment:
it's time to introduce lombok:

https://www.baeldung.com/intro-to-project-lombok

Reply:
I will create a ticket in the backlog, lets do this as a separate change